driver/ppm_fillers.c

```diff
@@ -1547,6 +1547,18 @@ int f_proc_startupdate(struct event_filler_arguments *args) {
 		/* Parameter 28: trusted_exepath (type: PT_FSPATH) */
 		res = val_to_ring(args, (unsigned long)trusted_exepath, 0, false, 0);
 		CHECK_RES(res);
+
+		/* Parameter 29: pgid (type: PT_UID) */
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(2, 6, 24)
```

Comment:
As in bpf/fillers.h, can we consider adding a small helper function since the same code is shared with `f_sched_prog_exec`?

Reply:
ty!